crypt.go

```diff
@@ -375,7 +375,8 @@ func activateWithTPMKey(tpm *TPMConnection, volumeName, sourceDevicePath, keyPat
 			if !lock {
 				return
 			}
-			lockErr = LockAccessToSealedKeys(tpm)
+			// TODO: Figure out what to do with this
+			//lockErr = LockAccessToSealedKeys(tpm)
```

Comment:
as I said I would drop this,  it would simplify this code and with the new function it's much easier to just to just do this immediately after the relevant activates for the consumer of the library